src/dom.js

```diff
@@ -544,3 +544,18 @@ export function escapeCssSelectorIdent(win, ident) {
   // Polyfill.
   return cssEscape(ident);
 }
+
+
+/**
+ * Returns a frame element if available.
+ * @param {!Window} win
+ * @return {?HTMLIFrameElement}
+ */
+export function getFrameElement(win) {
+  try {
+    return win.frameElement;
+  } catch (e) {
```

Comment:
In Safari this would yield a logged error independent of the catch. Can we instead only call the function when we know we are allowed to call it?